src/Modules/OrchardCore.Commerce/Extensions/RegionExtensions.cs

```diff
@@ -1,18 +1,14 @@
 using Microsoft.AspNetCore.Mvc.Rendering;
 using OrchardCore.Commerce.AddressDataType;
 using System.Collections.Generic;
-using System.Globalization;
 using System.Linq;
 
 namespace OrchardCore.Commerce.Extensions;
 
 public static class RegionExtensions
 {
-    public static IEnumerable<SelectListItem> CreateSelectListOptions(this IEnumerable<RegionInfo> regionInfos) =>
-        regionInfos.OrderBy(region => region.DisplayName).Select(region => new SelectListItem(
-            region.DisplayName,
+    public static IEnumerable<SelectListItem> CreateSelectListOptions(this IEnumerable<Region> regionInfos) =>
+        regionInfos.OrderBy(region => region.EnglishName).Select(region => new SelectListItem(
```

Comment:
I think `regionInfos` should be renamed to `regions` if you use the name `region` inside the `OrderBy` and `Select`.